components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -37,13 +37,13 @@ export default function RepositoryFinder(props: RepositoryFinderProps) {
             const result = [...suggestedContextURLs];
             try {
```

Comment:
```suggestion
            const result = [...suggestedContextURLs];
            searchString = searchString.trim();
            try {
```
To avoid having to trim in multiple places below.

Reply:
Resolved in [`ba6f09d`](https://github.com/gitpod-io/gitpod/pull/15561/commits/ba6f09d4da8fb328d8f72ca2aff0f1b303ef488f)